core/src/main/java/com/graphhopper/routing/weighting/PriorityWeighting.java

```diff
@@ -53,6 +52,9 @@ public double calcEdgeWeight(EdgeIteratorState edgeState, boolean reverse) {
         double weight = super.calcEdgeWeight(edgeState, reverse);
         if (Double.isInfinite(weight))
             return Double.POSITIVE_INFINITY;
-        return weight / (0.5 + edgeState.get(priorityEnc));
+        double priority = edgeState.get(priorityEnc);
+        if (priority > 1.5)
+            throw new IllegalArgumentException("priority cannot be bigger than 1.5 but was " + priority);
```

Comment:
Can we use `PriorityCode.getFactor(BEST.getValue())` also here?

Reply:
Sure